File: Scala/HelloWorld.scala
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._
import org.apache.spark.SparkConf

object HelloWorld{
        def main(args: Array[String]){
                val conf = new SparkConf().setAppName("Hello World")
                val sc = new SparkContext(conf)
                println("Hello World!")
        }
}

File: Java/HelloWorld.java
import org.apache.spark.api.java.*;
import org.apache.spark.SparkConf;

public class HelloWorld{
        public static void main(String[] args){
                SparkConf conf = new SparkConf().setAppName("Hello World");
                JavaSparkContext sc = new JavaSparkContext(conf);
                System.out.println("Hello World!");
        }
}
